lab_2/main.py

```diff
@@ -34,6 +55,14 @@ def get_language_features(language_profiles: dict) -> list or None:
     :param language_profiles: a dictionary of dictionaries - language profiles
     """
     pass
+    features = []
+    if (not isinstance(language_profiles,dict)) or (language_profiles == {}):
```

Comment:
`language_profiles == {}` is not a good practice - if you want to check whether a dictionary is empty or not, use `len`.